### ISSUE TITLE
Plugin cannot be used in non-Android application projects

### ISSUE THREAD

**Issue by fwcd** [jaredsburrows/gradle-license-plugin#148]
Using the plugin in a vanilla Gradle Java/Kotlin application (as generated by `gradle init`) like this:

```gradle
plugins {
    id 'application'
    id 'com.jaredsburrows.license' version '0.8.90'
}
```

results in the following error:

```
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find com.android.tools.build:gradle:3.6.0.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/com/android/tools/build/gradle/3.6.0/gradle-3.6.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project : > com.jaredsburrows.license:com.jaredsburrows.license.gradle.plugin:0.8.90 > com.jaredsburrows:gradle-license-plugin:0.8.90
```

This seems to be a regression, since plugin version `0.8.42` works fine.

**Comment by MartinAhrer** [comment]
I think you have to add the google repository.
```buildscript {
    repositories {
        gradlePluginPortal()
        mavenCentral()
        // required by com.jaredsburrows.license
        google()
    }
}```

**Comment by fwcd** [comment]
Thanks! Is there a way to use the plugin without pulling in the Android Build Tools?

**Comment by jaredsburrows** [comment]
I might need to make this `compileOnly` - https://github.com/KeepSafe/dexcount-gradle-plugin/blob/master/build.gradle#L77

**Comment by Averus89** [comment]
Hi I have the same problem as fwcd. 
```
plugins {
    (...)
    id "com.jaredsburrows.license" version "0.8.90"
}
```
I have added google() to the repositories below
```
repositories {
    mavenCentral()
    google()
    mavenLocal()
}
```
Still the same error about missing build tools. Changing to 0.8.42 is helping build project, however I have another exception while executing task
```
A problem was found with the configuration of task ':licenseReport' (type 'LicenseReportTask').
  - In plugin 'com.jaredsburrows.license' type 'com.jaredsburrows.license.LicenseReportTask' property 'productFlavors' annotated with @Internal should not be also annotated with @Optional.

```

**Comment by jaredsburrows** [comment]
@fwcd This is fixed in `0.9.0`.
